src/main/resources/swagger.api/mod-entities-links.yaml

```diff
@@ -239,6 +239,12 @@ components:
         linkingRuleId:
           type: integer
           description: ID of linking rule by which link should be created
+        status:
+          type: string
+          description: Status of the link. Read-only
+        errorCause:
+          type: string
+          description: Link update error cause (if present). Read-only
```

Comment:
You can put `readOnly` property instead mention it in the description. See docs: https://swagger.io/docs/specification/data-models/data-types/#readonly-writeonly